# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/WinWebAuthnService.cpp
# Commit: eb8cc8d7badf
# Full Hash: eb8cc8d7badf8643681a35d38d88c751e08926fa
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-11 09:01:30
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r add154b24d33 -r eb8cc8d7badf dom/webauthn/WinWebAuthnService.cpp
--- a/dom/webauthn/WinWebAuthnService.cpp	Mon Feb 10 20:11:06 2025 +0000
+++ b/dom/webauthn/WinWebAuthnService.cpp	Mon Feb 10 20:11:06 2025 +0000
@@ -406,7 +406,7 @@
           winRequireResidentKey = FALSE;
           winPreferResidentKey = FALSE;
         } else {
-          // WebAuthnManager::MakeCredential is supposed to assign one of the
+          // WebAuthnHandler::MakeCredential is supposed to assign one of the
           // above values, so this shouldn't happen.
           MOZ_ASSERT_UNREACHABLE();
           aPromise->Reject(NS_ERROR_DOM_UNKNOWN_ERR);